torch/optim/lr_scheduler.py

```diff
@@ -183,16 +187,21 @@ class MultiStepLR(_LRScheduler):
     """
 
     def __init__(self, optimizer, milestones, gamma=0.1, last_epoch=-1):
-        if not list(milestones) == sorted(milestones):
-            raise ValueError('Milestones should be a list of'
-                             ' increasing integers. Got {}', milestones)
+        milestones = set(milestones)
+        #if not list(milestones) == sorted(milestones):
+        #    raise ValueError('Milestones should be a list of'
+        #                     ' increasing integers. Got {}', milestones)
```

Comment:
What happens if a milestone shows up twice in the list?

Reply:
Good question. I actually think raising an error makes more sense, but the old codes decrease the rate twice at that milestone. Shall we make it BC?

Reply:
It doesn't seem... too hard to maintain BC, so might as well.

Reply:
I already made the updated version to maintain BC.